test/functional/Java8andUp/src/org/openj9/test/java/lang/Test_ClassLoader_SM.java

```diff
@@ -45,9 +45,8 @@ public void test_getParent1() {
 		System.setSecurityManager(manager);
 		try {
 			parentCl.getParent();
```

Comment:
The javadoc for the method should be updated to reflect this change.

What ensures that `parentCl` has no parent (so that an exception is not thrown)?